src/components/computed-properties.vue

```diff
@@ -259,18 +274,21 @@ export default {
       this.emptyForm();
       this.displayList = false;
     },
-    validateDuplicate() {
-      this.existsProperty = false;
-      this.current.forEach(item => {
-        if (item.property === this.add.property && item.id !== this.add.id) {
-          this.existsProperty = true;
+    validateData() {
+      this.ruleDescription = 'required';
+      this.ruleFormula = 'required';
+      this.rulePropName = this.rulePropName ? this.rulePropName : 'required';
```

Comment:
A ternary here isn't necessary; I think you can make it more readable by using a logical OR operator:
```js
this.rulePropName = this.rulePropName || 'required';
```

Reply:
fixed